src/openms/source/FORMAT/IdXMLFile.cpp

```diff
@@ -101,13 +101,13 @@ namespace OpenMS
 
     //write header
     os << "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n";
-    os << "<?xml-stylesheet type=\"text/xsl\" href=\"http://open-ms.sourceforge.net/XSL/IdXML.xsl\" ?>\n";
+    os << "<?xml-stylesheet type=\"text/xsl\" href=\"https://raw.githubusercontent.com/OpenMS/OpenMS/develop/share/OpenMS/XSL/IdXML.xsl\" ?>\n";
```

Comment:
Both is kind of suboptimal.
I would prefer a stable URL pointing to: www.openms.org/...
but I don't know how well redirects are handled or allowed.
also  the XSL should contain the version number of the idXML version it corresponds to.